marketdata/stream/client.go

```diff
@@ -53,6 +53,10 @@ type StocksClient interface {
 	UnsubscribeFromStatuses(symbols ...string) error
 	SubscribeToLULDs(handler func(luld LULD), symbols ...string) error
 	UnsubscribeFromLULDs(symbols ...string) error
+	RegisterHandlerCancelErrors(handler func(tce TradeCancelError))
```

Comment:
[minor] Maybe I would skip the "Handler" from the method name. Not completely sure though.

Reply:
:thinking: Removed to make it cleaner